lib/helpers/model/castBulkWrite.js

```diff
@@ -117,8 +118,9 @@ module.exports = function castBulkWrite(originalModel, op, options) {
           const updatedAt = model.schema.$timestamps.updatedAt;
           applyTimestampsToUpdate(now, createdAt, updatedAt, op['updateMany']['update'], {});
         }
-
-        applyTimestampsToChildren(now, op['updateMany']['update'], model.schema);
+        if (op['updateMany'].timestamps || op['updateMany'].timestamps == null) {
```

Comment:
Nitpick:
```suggestion
        if (op['updateMany'].timestamps === true || op['updateMany'].timestamps == null) {
```

Reply:
Can we do `if (op['updateMany'].timestamps !== false)` to be consistent with the other `if` statements in this file?